### PR TITLE
[11.0][FIX] account_banking_sepa_direct_debit: printing report

### PR DESCRIPTION
**Pull request by astirpe**
Fixes https://github.com/OCA/bank-payment/issues/439

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please change module version number

**Comment by astirpe** [comment]
@pedrobaeza done

File: account_banking_sepa_direct_debit/reports/sepa_direct_debit_mandate.xml [hunk 1]
```diff
@@ -1,6 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
+    <template id="assets_common" name="account_banking_sepa_direct_debit pdf assets" inherit_id="web.report_assets_common">
```
Comment (pedrobaeza) [inline account_banking_sepa_direct_debit/reports/sepa_direct_debit_mandate.xml:4]: With this, you are overwriting the style for all the reports. In-line style is not working? If you want to do it this way, please put all the styles under a class for not changing general ones.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: account_banking_sepa_direct_debit/reports/sepa_direct_debit_mandate.xml (hunk 1 above)
Comment (astirpe) [reply account_banking_sepa_direct_debit/reports/sepa_direct_debit_mandate.xml:4]: @pedrobaeza you're right! I made the changes, could you check?

**Review by pedrobaeza — APPROVED** [review]
As this is broken, I fast-track it for fixing it

**Pull request merged**